src/App.jsx

```diff
@@ -2,38 +2,52 @@ import { FaHome } from "react-icons/fa";
 import { MdOutlineAccountCircle } from "react-icons/md";
 import { PiCookingPotBold } from "react-icons/pi";
 import HomePage from "./Components/HomePage";
-
+import { BrowserRouter, Routes, Route, Link } from "react-router-dom";
+import AddRecipePage from "../pages/AddRecipePage";
+import Profile from "./Components/Profile";
 function App() {
   return (
-    <div className="flex">
-      <aside className="flex h-screen w-72 bg-white justify-center">
-        <div className="flex flex-col justify-between ">
-          <div className=" ">
-            <img
-              src="../src/assets/output-onlinepngtools.png"
-              alt="cooking logo"
-              className="w-30 h-20"
-            />
-          </div>
-          <div className="flex flex-col gap-5">
-            <div className="flex flex-row">
-              <FaHome className="inline text-lg mb-1 mr-1 h-6 w-6" />
-              <div className="text-md">Home</div>
-            </div>
-            <div className="flex flex-row">
-              <MdOutlineAccountCircle className="inline text-lg mb-1 mr-1 h-6 w-6" />
-              <div className="text-md">Profile</div>
+    <BrowserRouter>
+      <div className="flex ">
```

Comment:
Move the SideMenu into a seperate component, create folder inside the components folder named nav
Path: components/nav/SideMenu.tsx so we divide components who are specific to navbar and reusable components throughout the application